apps/indexer/ponder.schema.ts

```diff
@@ -124,16 +128,15 @@ export const transfer = onchainTable(
   }),
   (table) => ({
     pk: primaryKey({
-      columns: [table.transactionHash],
+      columns: [table.transactionHash, table.fromAccountId, table.toAccountId],
     }),
-    transfersTokenIdx: index().on(table.tokenId),
```

Comment:
Similar relation issue as above. Matching `votingPowerHistory.deltaMod` with `delegation.delegatedValue` creates an invalid foreign key relationship since these fields don't represent the same data.